mod_pbxproj/mod_pbxproj.py

```diff
@@ -719,11 +798,16 @@ def get_groups_by_os_path(self, path):
 
         return groups
 
-    def get_build_phases(self, phase_name):
-        phases = [p for p in self.objects.values() if p.get('isa') == phase_name]
+    def get_build_phases(self, phase_isa, phase_name = None):
```

Comment:
this changes breaks the backwards compatibility for anyone that's currently using the script. Must be reverted and update the new code to make use of the new functionality